tests/test_analysis/test_feature_relevance/test_relevance_table.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.tree import DecisionTreeRegressor
+
+from etna.analysis.feature_relevance import get_model_relevance_table
+from etna.analysis.feature_relevance import get_statistics_relevance_table
+
+
+@pytest.mark.parametrize(
+    "method,method_kwargs",
+    ((get_statistics_relevance_table, {}), (get_model_relevance_table, {"model": DecisionTreeRegressor()})),
+)
+def test_interface(method, method_kwargs, simple_df_relevance):
+    relevance_table = method(simple_df_relevance[0], simple_df_relevance[1], **method_kwargs)
+    assert isinstance(relevance_table, pd.DataFrame)
+    assert sorted(relevance_table.index) == sorted(
+        simple_df_relevance[0].columns.get_level_values("segment").unique().tolist()
```

Comment:
I know it has been the case before, but tolist() is not necessary with sorted()
```suggestion
        simple_df_relevance[0].columns.get_level_values("segment").unique()
```